src/Cli/dotnet/NativeMethods.txt

```diff
@@ -1,4 +1,6 @@
-﻿// Methods
+﻿// This file is used by Microsoft.Windows.CsWin32 to generate all the native structs and API calls.
```

Comment:
I added this comment here because I didn't know what this file did just looking at it. So, this'll let someone else know what it does!

Reply:
@joeloff has talked to me about this file and I assume he understands it. I would hope not, but might be good to verify with him there is no dependency somewhere on the comment that says 'Methods'

Reply:
I talked to him and he's the one that said it was fine to add a comment. `// Methods` wasn't removed; it is just two lines below this now.